modules/administration-guide/partials/proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards.adoc

```diff
@@ -1,31 +1,38 @@
 [id="proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards"]
-= Viewing {devworkspace} Operator metrics on Grafana dashboards
+= Viewing {devworkspace} Operator metrics from an OpenShift web console dashboard
 
-To view the {devworkspace} Operator metrics on Grafana with the example dashboard:
+After configuring the in-cluster Prometheus instance to collect {devworkspace} Operator metrics, you can view the metrics on a custom dashboard in the *Administrator* perspective of the OpenShift web console.
 
 .Prerequisites
 
-* Prometheus is collecting metrics. See xref:proc_collecting-dev-workspace-operator-metrics-with-prometheus[].
-
-* Grafana version 7.5.3 or later.
-
-* Grafana is running on port `3000` with a corresponding Service. See link:https://grafana.com/docs/grafana/latest/installation/kubernetes/[Installing Grafana].
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 
+* An instance of {prod-short} running in {orch-name}.
```

Comment:
Since this section is only applicable to OpenShift and not vanilla Kubernetes, adding this suggestion to hardcode the attribute/variable values because the upstream docs show Kubernetes:
```suggestion
* Your organization's instance of {prod-short} is installed and running in Red Hat OpenShift.

* An active `oc` session with administrative permissions to the destination OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the CLI].
```